SurveyApp/Modules/Login/LoginViewController.swift

```diff
@@ -33,15 +43,127 @@ final class LoginViewController: UIViewController {
 // MARK: - LoginViewInput
 
 extension LoginViewController: LoginViewInput {
-
+    
     func configure() {
+        setUpLayout()
         setUpViews()
     }
 }
 
+// MARK: - Private functions
+
 extension LoginViewController {
     
+    private func setUpLayout() {
+        overlayView.layer.insertSublayer(gradientLayer, at: 0)
+
+        view.addSubview(backgroundImageView)
+        view.addSubview(blurEffectView)
+        view.addSubview(overlayView)
+        view.addSubview(logoImageView)
+        view.addSubview(emailField)
+        view.addSubview(passwordField)
+        view.addSubview(forgotButton)
+        view.addSubview(loginButton)
+
+        backgroundImageView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+            $0.width.equalTo(UIScreen.main.bounds.width)
+        }
+
+        logoImageView.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(150.0)
+            $0.centerX.equalToSuperview()
+            $0.height.equalTo(40.0)
+            $0.width.equalTo(180.0)
+        }
+
+        emailField.snp.makeConstraints {
+            $0.top.equalTo(logoImageView.snp.bottom).offset(120.0)
+            $0.leading.trailing.equalToSuperview().inset(24.0)
+            $0.height.equalTo(55.0)
+        }
+
+        passwordField.snp.makeConstraints {
+            $0.top.equalTo(emailField.snp.bottom).offset(20.0)
+            $0.leading.trailing.equalToSuperview().inset(24.0)
+            $0.height.equalTo(55.0)
+        }
+
+        forgotButton.snp.makeConstraints {
+            $0.top.equalTo(emailField.snp.bottom).offset(20.0)
+            $0.trailing.equalTo(passwordField.snp.trailing)
+            $0.leading.equalToSuperview().inset(280.0)
+            $0.height.equalTo(55.0)
+        }
+
+        loginButton.snp.makeConstraints {
+            $0.top.equalTo(passwordField.snp.bottom).offset(20.0)
+            $0.leading.trailing.equalToSuperview().inset(24.0)
+            $0.height.equalTo(55.0)
+        }
+    }
+    
     private func setUpViews() {
-        view.backgroundColor = .white
+        let backgroundImage = R.image.loginBackground()
```

Comment:
we usually have a `R.swift+Typealias.swift` file where we define type aliases like `typealias Asset = R.image`
then you can have `let backgroundImage = Asset.loginBackground()`